tests/cyclers/38-unittests.t

```diff
@@ -1,25 +0,0 @@
-#!/bin/bash
-# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
-# Copyright (C) 2008-2019 NIWA & British Crown (Met Office) & Contributors.
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#-------------------------------------------------------------------------------
-# Test intercycle dependencies.
-. "$(dirname "$0")/test_header"
-#-------------------------------------------------------------------------------
-set_test_number 3
-#-------------------------------------------------------------------------------
-run_ok "${TEST_NAME_BASE}-cycling-iso8601" python3 -m 'cylc.flow.cycling.iso8601'
-run_ok "${TEST_NAME_BASE}-cycling-integer" python3 -m 'cylc.flow.cycling.integer'
-run_ok "${TEST_NAME_BASE}-cycling" python3 -c 'import cylc.flow.cycling'
```

Comment:
Huh, I guess this used to actually run some primitive unit tests in the source modules, and we forgot to delete it once we started doing unit tests separately.